### ISSUE TITLE
Accessibility: Focus treatment missing in IE/Edge

### ISSUE THREAD

**Issue by JllyGrnGiant** [highcharts/highcharts#6907]
#### Live demo with steps to reproduce
https://codepen.io/anon/pen/dRZLNo

1. Click in the render area to give focus to the iframe
2. Tab through the chart and legend

#### Expected behaviour
Screenshots taken from Chrome
![image](https://user-images.githubusercontent.com/1325800/27614980-e00d0862-5b59-11e7-882d-c2d27e9db1c9.png)
![image](https://user-images.githubusercontent.com/1325800/27614990-f20cba80-5b59-11e7-8f19-98dbc9e00c28.png)

#### Actual behaviour
No focus borders making it difficult to tell what pie slice you're focused on and impossible for what legend item you're focused on.

#### Affected browsers
- IE
- Edge

**Comment by sebastianbochan** [comment]
Hi @JllyGrnGiant,
Thank you for the reporting.

@oysteinmoseng
